examples/interactive_play.py

```diff
@@ -252,6 +262,42 @@ def get_input_vel_ctlr(
             logger.info("[play.py]: Snapping")
             magic_grasp = 1
 
+    if use_joint_control:
+        # Add random noise to human arms but keep global transform
+        (
+            new_joints,
+            root_trans,
+        ) = env._sim.articulated_agent.get_joint_transform()
+        num_joints = len(new_joints) // 4
```

Comment:
where does the 4 come from? Can you create a variable with value 4 with a nice name that describes this magic number?

Reply:
A variable is a bit overkill since it corresponds to 4 dimensions per quaternion, and is used in the subsequent indexing. I added a comment right there.

Reply:
Hm I don't see this comment. Are the latest changes pushed to the branch?